pkg/ruler/ruler.go

```diff
@@ -509,7 +509,7 @@ func (r *Ruler) syncManager(ctx context.Context, user string, groups store.RuleG
 				level.Error(r.logger).Log("msg", "unable to create rule manager", "user", user, "err", err)
 				return
 			}
-			manager.Run()
+			go manager.Run()
```

Comment:
This is now a blocking call

Reply:
Would you mind adding a code comment to explain that the manager `Run()` start running the manager and blocks until the `Stop()` is called, please?

Reply:
Done